mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go

```diff
@@ -14,8 +14,26 @@ import (
 	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
 )
 
+func TestAccSecurityCenterManagement(t *testing.T) {
```

Comment:
After checking the issue, it looks like `testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule` and `testAccSecurityCenterManagementOrganizationSecurityHealthAnalyticsCustomModule` cannot be run at the same time. These two tests need to be run in serialization and put into the testCases below on line 18.